.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
- [ ] Great job adding a `.gitignore` file, and including `node_modules` to the file as well, please also add `test.md` into the file as it is required. For this project your `.gitignore` should have two files, for example:

<kbd>
<img width="657" alt="Screenshot 2023-04-26 at 7 33 34 AM" src="https://user-images.githubusercontent.com/44798044/234451878-c7a69b0c-6409-449f-917d-bb4b98d84e4c.png">
</kbd>

- [ ] [OPTIONAL] You should also include `.DS store` file into the `.gitignore`:

 <kbd>
<img width="390" alt="Screenshot 2023-04-26 at 8 21 59 AM" src="https://user-images.githubusercontent.com/44798044/234461378-f8783755-04fe-4311-bdd2-c97232d92610.png">
</kbd>